### ISSUE TITLE
mx-maskrcnn for keypoint detection

### ISSUE THREAD

**Issue by tianyuecao** [TuSimple/mx-maskrcnn#144]
I would like to use maskrcnn to do keypoint detection, what should i change?

any suggestions are welcome.

**Comment by tianyuecao** [comment]
I've modified this project by referring [a tensorflow version](https://github.com/Superlee506/Mask_RCNN_Humanpose)  but find the keypoint softmax loss hard to converge. Can anyone help me please?
